flake.nix

```diff
@@ -61,12 +84,23 @@
     in
     {
       homeConfigurations = {
+        "jente@aarch64-darwin" = mkHomeConfig {
+          username = "jente";
+          system = "aarch64-darwin";
+        };
         "jente@x86_64-linux" = mkHomeConfig {
           username = "jente";
           system = "x86_64-linux";
         };
       };
 
+      darwinConfigurations = {
+        "jente@aarch64-darwin" = mkDarwinConfig {
+          system = "aarch64-darwin";
+          # extraModules = [ ./profiles/personal.nix ./modules/darwin/apps.nix ];
```

Comment:
Sort this out with work-specific stuff and how to install (graphical) applications such as Spotify.